src/action.py

```diff
@@ -216,9 +238,12 @@ def make_actor(say):
     actor.add_keyword(_('repeat after me'),
                       RepeatAfterMe(say, _('repeat after me')))
 
-    # =========================================
-    # Makers! Add your own voice commands here.
-    # =========================================
+# =========================================
+# Makers! Add your own voice commands here.
+# =========================================
+
+    actor.add_keyword(_('power off'), PowerCommand(say, 'shutdown'))
+    actor.add_keyword(_('reboot'), PowerCommand(say, 'reboot'))
```

Comment:
I am concerned that this might trigger in a search "How to I reboot my laptop" :/